docs/content/go-components.md

```diff
@@ -132,7 +132,7 @@ allowed_http_hosts = [ "https://some-random-api.ml" ]
 route = "/hello"
 ```
 
-> Spin HTTP components written in Go must currently use the Wagi executor.
+> Spin HTTP components written in Go can now use the Spin executor.
```

Comment:
nit: should probably drop the "now" (or even lose the para altogether?) - a reader not already familiar with the "old" way doesn't need to know that it used to be different

Reply:
Fixed.